Theme.ts

```diff
@@ -140,8 +143,8 @@ export const GlobalStyle = createGlobalStyle`
   //
 
   body {
-    background-color: ${(props) => props.theme.colours.purple};
-    color: ${(props) => props.theme.colours.cream};
+    background-color: ${(props) => props.theme.colours.white};
+    color: ${(props) => props.theme.colours.grey};
```

Comment:
I've used brand plum as font colour. It seems to work ok. Is this grey what they use on the point site or did you add it?

Reply:
I took it from the website, and cleaned up some of the other colours from there that weren't in use.